.github/workflows/format.yml

```diff
@@ -0,0 +1,20 @@
+name: Rustfmt
+
+on:
+  # Triggers the workflow on push request events but only for the master branch
+  push:
+    branches: [ master ]
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            components: rustfmt
+            override: true
+      - uses: mbrobbel/rustfmt-check@master
```

Comment:
Suggestion: I'd prefer not to rely on a small third-party CI job, as these can be sources of breakage and security issues. In ICU4X we built out the CI to to run the command manually. There is a bit of misdirection in the following links around the use of cargo make which I think you could ignore and put the format command directly in the yml file, but rely on the rustup commands.

https://github.com/unicode-org/icu4x/blob/d418731826aea10cbc72d6a3b73789a981e425bd/.github/workflows/build-test.yml#L513-L550

https://github.com/unicode-org/icu4x/blob/d418731826aea10cbc72d6a3b73789a981e425bd/tools/scripts/tidy.toml#L14-L18

Reply:
I've removed the dependency on `mbrobbel/rustfmt-check`. There might be an argument about including the fmt check step with the `test.yml` actions as it also needs the rust toolchain download/install but imo it's cleaner if it's a separate job.